.eslintrc

```diff
@@ -59,6 +59,7 @@ rules:
   indent: [2, 2, {SwitchCase: 1}]
   key-spacing: [2, {mode: "minimum"}]
   keyword-spacing: 2
+  linebreak-style: 2
```

Comment:
`"unix"` is the default so just enabling it with `2` is enough. However, if explicitness is desired, I'm happy to change it to `[2, "unix"]`.